astropy/cosmology/core.py

```diff
@@ -169,12 +169,12 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=0, Neff=3.04,
         self.name = name
 
         # Tcmb may have units
-        self._Tcmb0 = u.Quantity(Tcmb0, unit=u.K, dtype=np.float)
+        self._Tcmb0 = u.Quantity(Tcmb0, unit=u.K, dtype=float)
```

Comment:
This can be removed altogether (the default is `float64`, which is what one would want here).